src/main/java/uk/gov/hmcts/reform/blobrouter/services/email/EmailSender.java

```diff
@@ -4,11 +4,13 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.mail.javamail.JavaMailSender;
 import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
 
 import java.io.File;
 import java.util.Map;
 import javax.mail.internet.MimeMessage;
 
+@Component
 public class EmailSender {
```

Comment:
Missing condition on `spring.mail.hostname`

Reply:
This condition will be in the class using this EmailSender.

Reply:
But the component will be built. Or at least tried to. Since it has dependency of `JavaMailSender` it'll fail due to `spring.mail.hostname=false` therefore - condition is required to be propagated upstream to all dependants